src/Client/Dialogs/ReportDialog.razor.cs

```diff
@@ -54,7 +57,16 @@ protected override Task OnInitializedAsync()
     }
 
     protected async Task CopyToClipboard()
-        => await ClipboardService!.WriteTextAsync(ReportText ?? string.Empty);
+    {
+        if (Snackbar is null)
+        {
+            throw new InvalidOperationException(nameof(Snackbar));
+        }
+
+        await ClipboardService!.WriteTextAsync(ReportText ?? string.Empty);
```

Comment:
Defaulting to `string.Empty` instead of a `null` string parameter, should probably not be a responsibility of the caller.

But that's a minor API change. Nothing to block this from merging. ☺️

Reply:
Also, rather than asserting `ClipboardService` is not null here, you'd be better off checking it or safely calling it as:
```cs
await ClipboardService?.WriteTextAsync(ReportText ?? string.Empty);
```

But again, that's just a minor point.